src/shaders/fill_extrusion.vertex.glsl

```diff
@@ -6,7 +6,7 @@ uniform float u_vertical_gradient;
 uniform lowp float u_opacity;
 
 attribute vec4 a_pos_normal_ed;
-attribute vec2 a_centroid_pos;
+attribute vec2 aCentroidPos;
```

Comment:
I made this change to keep the linter happy.
Since we also need to read it on the CPU side now and the linter was not happy with non-camel-case property `.a_centroid_pos0`

Reply:
I'd go for code consistency here — it's weird to see the naming mixed. Disabling linter rules on a line-by-line basis is fine.